modules/accounting/includes/functions/products.php

```diff
@@ -117,6 +118,19 @@ function erp_acct_insert_product( $data ) {
         $wpdb->query( 'START TRANSACTION' );
         $product_data = erp_acct_get_formatted_product_data( $data );
 
+        $product_check =  $wpdb->get_row(
+            $wpdb->prepare(
+                "SELECT * FROM {$wpdb->prefix}erp_acct_products where name = %s",
+                $product_data['name']
+            ),
+            OBJECT
+        );
+
+       if ( $product_check ) {
+            throw new \Exception( __( $product_data['name']." product already exists!" ) ) ;
```

Comment:
You missed text domain 👍 
anyways change this line to:
`throw new \Exception( $product_data['name'] . ' ' .  __( "product already exists!" , 'erp) ) ;`

Reply:
Done